docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -44,6 +44,16 @@ Then, use Kustomize to apply `bundle.yaml` with the changed namespace.
 kubectl apply -k .
 ----
 
+If you choose two deploy multiple Hazelcast Operators in different namespaces, you will need to update the name of every operator. You can add `namePrefix` field in the `kustomization.yaml` to differentiate them.
+
+[source,yaml]
+----
+namespace: <your-namespace>
+resources:
+- bundle.yaml
+namePrefix: <operator-prefix>
```

Comment:
I think suffix is better for readability and follow-up.
https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/README.md#prefixsuffix-transformer

Reply:
Sure I will change it